src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -40,10 +42,13 @@ internal class SonarQubeServiceWrapper : ISonarQubeServiceWrapper
         public const string ProjectDashboardRelativeUrl = "dashboard/index/{0}";
 
         public const string RoslynExporter = "roslyn-cs";
-
-        public const string CSharpLanguage = "cs";
-        public const string VBLanguage = "vbnet";
-        public static readonly string[] SupportedLanguages = { CSharpLanguage, VBLanguage };
+        
+        private static readonly IReadOnlyDictionary<Language, string> languageKeys = new ReadOnlyDictionary<Language, string>(
```

Comment:
I think it's safe to assume the language keys are stable - changing one would break far more than just this feature.